crates/sargon-uniffi/src/profile/mfa/security_structures/security_shield_builder.rs

```diff
@@ -252,6 +252,29 @@ impl SecurityShieldBuilder {
 
 #[uniffi::export]
 impl SecurityShieldBuilder {
+    /// "Statically" queries which FactorSourceKinds are disallowed for authentication signing.
```

Comment:
"statically" put in quoutes since UniFFI does not allow static methods to be exported.

Documented here so that one understands that this does not mutate any state.